scripts/chocolatey/build/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,230 @@
+<#
+.SYNOPSIS
+Installs the Lacework CLI.
+
+Authors: Technology Alliances <tech-ally@lacework.net>
+
+.DESCRIPTION
+This script installs the Lacework Command Line Interface.
+
+#>
+
+$ErrorActionPreference = "stop"
+
+Set-Variable GithubReleasesRootUrl -Option ReadOnly -value "https://github.com/lacework/go-sdk/releases" -Force
+Set-Variable PackageName -Option ReadOnly -value "lacework-cli-windows-amd64.exe.zip" -Force
```

Comment:
I notice this is assuming `amd64` do we need to add any consideration for any different arch types?

Reply:
Our existing installer doesn't